examples/jsm/renderers/nodes/functions/BSDFunctions.js

```diff
@@ -0,0 +1,125 @@
+import { PI, RECIPROCAL_PI } from '../consts/MathConsts.js';
```

Comment:
Minor typo: `BSDFFunctions.js `

Reply:
I thought it would be a bit redundant a extra function? 
BSDFFfunctions -> `Bidirectional Scattering Distribution Function Functions`

Reply:
Maybe `BSDFs.js` ?

Reply:
Good point. `BSDFs.js` sounds good to me!